racingcar-java/src/test/java/SetTest.java

```diff
@@ -39,11 +39,9 @@ void containsTest(int number) {
     @CsvSource(value = {"1:true", "2:true", "3:true", "4:false", "5:false"}, delimiter = ':')
     @DisplayName("존재/존재하지 않는 값 테스트")
     void containsVarietyTest(String input, String expected) {
```

Comment:
```suggestion
    void containsVarietyTest(int input, boolean expected) {
```

CSV 를 사용하더라도, 파라미터 타입을 int로 하면 자동으로 형변환 해주는걸로 알아요
이부분 참고하세요!

Reply:
감사합니다! 코드가 너무 쓸데없이 복잡했었네요